src/eth/Web3Service.js

```diff
@@ -313,7 +275,6 @@ export default class Web3Service extends PrivateService {
     });
 
     this.manager().onDeauthenticated(() => {
-      this._removeBlockUpdates();
```

Comment:
@sirromdev because this shouldn't have been added--notice that the [code it's replacing](https://github.com/makerdao/dai.js/pull/73/files#diff-785cfb5527ab18b80ae673e64d1947ceL200) only sets up `onDisconnected`, not `onDeauthenticated`. so this is a case of one bug hiding another bug... not fun to diagnose

Reply:
ping me in chat if it'd be helpful for me to explain over video